ddlambda.go

```diff
@@ -35,6 +36,9 @@ type (
 		// Site is the host to send metrics to. If empty, this value is read from the 'DD_SITE' environment variable, or if that is empty
 		// will default to 'datadoghq.com'.
 		Site string
+
+		// DebugLogging will turn on extended debug logging.
+		DebugLogging bool
```

Comment:
Do you think it's worth making it configurable through the environment variable `DD_DEBUG` as well if not difficult?